src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -28,6 +28,12 @@
             <tag name="monolog.logger" channel="translation" />
         </service>
 
+        <service id="translator.formatter" alias="translator.formatter.default" />
```

Comment:
Do we need this? I mean this is just redundant with what we do in the extension (in combination with the default option value).

Reply:
Oh, in fact there is one difference: The alias registered in the extension is private. So IMO that's a good argument to remove this one here. At least it will avoid confusion when reading the code.

Reply:
(this one also is private, all file start with public="false" as default ;) )

Reply:
hm okay 😆

Reply:
the default value can be overridden throughout the config, without using the alias we can't inject the default one if we want.

Reply:
But we always create the alias in https://github.com/aitboudad/symfony/blob/fd9f484879a08e00a42a3888d8b1681923b8eb0b/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1078, don't we (`$config['formatter']` will have the default value if the user didn't provide any)?

Reply:
you're right, removed!